ssms/hssm_support.py

```diff
@@ -501,11 +499,4 @@ def rng_fn(
         shape_spec = _reshape_sims_out(max_shape, n_replicas, obs_dim_int)
         sims_out = sims_out.reshape(shape_spec)
 
-    sims_out = apply_lapse_model(
-        sims_out=sims_out,
-        p_outlier=p_outlier,
-        rng=rng,
-        lapse_dist=cls._lapse,
-        choices=choices,
-    )
-    return sims_out
+    return sims_out, p_outlier
```

Comment:
Only qualm I am having here is that `p_outlier` shouldn't even occur at all tbh. 
I would say we keep `_get_p_outlier()` in `hssm`, and call it before the call to the simulator.

Reply:
(sidenote): I think the simulators are robust to the parameter being passed unused, but that would probably be a bit too hacky.